ts/Core/Series/Point.ts

```diff
@@ -1301,33 +1292,45 @@ class Point {
     }
 
     /**
-     * Import events from the series' and point's options. Only do it on
+     * Manage specific event from the series' and point's options. Only do it on
      * demand, to save processing time on hovering.
      *
      * @private
      * @function Highcharts.Point#importEvents
      */
-    public importEvents(): void {
-        if (!this.hasImportedEvents) {
-            const point = this,
-                options = merge(
-                    point.series.options.point as PointOptions,
-                    point.options
-                ),
-                events = options.events;
-
-            point.events = events;
+    public manageEvent(eventType: string): void {
+        const point = this,
+            options = merge(
+                point.series.options.point as PointOptions,
+                point.options
```

Comment:
Shouldn't `point.options` already include `SeriesOptions.point` when instantiated? Or is this about `Series.update` follow-ups?

Reply:
Only `point.options` doesn't contain every events, but `point.series.options.point` indeed contains everything. I changed that part of the code a little bit.